llama-index-integrations/embeddings/llama-index-embeddings-nvidia/llama_index/embeddings/nvidia/base.py

```diff
@@ -14,12 +14,17 @@
 
 from openai import OpenAI, AsyncOpenAI
 
-BASE_URL = "https://ai.api.nvidia.com/v1/retrieval/nvidia/"
+# integrate.api.nvidia.com is the default url for most models, any
+# bespoke endpoints will need to be added to the MODEL_ENDPOINT_MAP
+BASE_URL = "https://integrate.api.nvidia.com/v1/"
 DEFAULT_MODEL = "NV-Embed-QA"
 
+# because MODEL_ENDPOINT_MAP is used to construct KNOWN_URLS, we need to
+# include at least one model w/ https://integrate.api.nvidia.com/v1/
 MODEL_ENDPOINT_MAP = {
-    DEFAULT_MODEL: BASE_URL,
+    "NV-Embed-QA": "https://ai.api.nvidia.com/v1/retrieval/nvidia/",
```

Comment:
is NV-Embed-QA  an embedding model?

Reply:
it is, see https://build.nvidia.com/nvidia/embed-qa-4?snippet_tab=Shell

Reply:
Cool. Thanks.